cmd/oci/oci.go

```diff
@@ -38,6 +38,8 @@ func init() {
 	OCICmd.Flags().BoolVarP(&dfSwap, "df-swap", "", false, "Modify base images in Dockerfile")
 	OCICmd.Flags().StringVarP(&tag, "tag", "t", "", "The tag that will be replaced with original tag in Dockerfile")
 	OCICmd.Flags().StringVar(&path, "path", "", "The path to Dockerfile")
+	OCICmd.Flags().BoolVar(&digest, "digest", false, "Get the digest of the image")
+	OCICmd.Flags().StringVar(&auth, "auth", "", "The auth to use to authenticate to the registry")
```

Comment:
Let's use the same flag name as skopeo for now so this should be `dest-creds`. 
Also, please add an example on how to use it in the `Long` section(for both auth and digest).